package.json

```diff
@@ -19,28 +19,28 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "6.0.0-rc.5",
-    "@angular/cdk": "6.0.0-rc.13",
-    "@angular/common": "6.0.0-rc.5",
-    "@angular/compiler": "6.0.0-rc.5",
-    "@angular/core": "6.0.0-rc.5",
-    "@angular/forms": "6.0.0-rc.5",
-    "@angular/http": "6.0.0-rc.5",
-    "@angular/material": "6.0.0-rc.13",
+    "@angular/animations": "6.0.0-rc.6",
+    "@angular/cdk": "6.0.0-rc.14",
+    "@angular/common": "6.0.0-rc.6",
+    "@angular/compiler": "6.0.0-rc.6",
+    "@angular/core": "6.0.0-rc.6",
+    "@angular/forms": "6.0.0-rc.6",
+    "@angular/http": "6.0.0-rc.6",
+    "@angular/material": "6.0.0-rc.14",
     "@angular/material-moment-adapter": "6.0.0-rc.0",
```

Comment:
`@angular/material-moment-adapter` should probably be the same version as the CDK/Material.